OSX/README.md

```diff
@@ -27,12 +22,7 @@ TIP: Use `touch AGS.app` to force macos to reload the application icon.
 
 ## Adding resources for the game
 
-All game resources are taken from `OSX/Resources` . Please feel free to replace the stub files that
-exist there with your own game files, as long as they have the same names.
-
-The default accepted name for the game file to be placed there is `game.ags`, which is generated in the `Compiled/Data/`
```

Comment:
Please, keep a mention of `Compiled/Data/` folder. I think it should say that normally game files are taken from Compiled/Data (where *.ags is a main game data file), and more files could be added if necessary for mac version of the game.

Reply:
Added the following sentence, what do you think?

> When you build your game with the Editor, your own game `.ags` and additional files should be in `Compiled/Data/`.

Reply:
My impression from this sentence is that it does not make it clear that the files are taken *from* there, it sounds more like user is supposed to place them there themselves.

Reply:
switched the order of the sentences to be more obvious, do you think it works better like this?

> After you build your game with the Editor, your own game `.ags` and additional files will be generated be in the `Compiled/Data/` of your game project directory.
> 
> In this port, all game resources are taken from `OSX/Resources`, remove the stub files that exist there and replace with your own game files taken from `Compiled/Data/`.

Reply:
This may be fine; there's just one typo in the sentence "files will be generated **be** in the"

Reply:
thanks! Took the opportunity to fix some other grammar errors I haven't thought about to check!